### PR TITLE
MM-25111 Replace usage of LocalizedIcon in 'channel_members_rhs/header.tsx' with i/span tags

### PR DESCRIPTION
**Pull request by Vinecreeper888**
Removed LocalizedIcon in header.tsx
Replaced it with <i/> tag and used useIntl hook of react-intl for the aria label in header.tsx
Resolves https://github.com/mattermost/mattermost/issues/25111

Screenshots
BEFORE
old

AFTER
new

https://mattermost.atlassian.net/browse/MM-55099

```release-note
NONE
```

### CODE REVIEW

**Comment by M-ZubairAhmed** [comment]
@Vinecreeper888 can you please check the failing CI

File: webapp/channels/src/components/channel_members_rhs/header.tsx [hunk 1]
```diff
@@ -85,9 +83,12 @@ const Header = ({channel, canGoBack, onClose, goBack}: Props) => {
                     aria-label='Close'
                     onClick={onClose}
                 >
-                    <LocalizedIcon
+                    <i
                         className='icon icon-close'
-                        ariaLabel={{id: t('rhs_header.closeTooltip.icon'), defaultMessage: 'Close Sidebar Icon'}}
+                        aria-label={useIntl().formatMessage({
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/channel_members_rhs/header.tsx:88]: please define hook seperately

```
    const {formatMessage} = useIntl();
   ```

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]
Please check the failing CI

File: webapp/channels/src/components/channel_members_rhs/header.tsx (hunk 1 above)
Comment (Vinecreeper888) [reply webapp/channels/src/components/channel_members_rhs/header.tsx]: has been resolved

**Comment by Vinecreeper888** [comment]
@M-ZubairAhmed how do I check the Enterprise CI/tests that have failed?

**Comment by M-ZubairAhmed** [comment]
> @M-ZubairAhmed how do I check the Enterprise CI/tests that have failed?

Sorry these cannot be checked without correct permissions. I have updated to latest branch to see if it solves that issue

**Review by M-ZubairAhmed — APPROVED** [review]
Looks great @Vinecreeper888 thank you for your contribution

**Review by yasserfaraazkhan — APPROVED** [review]

**Pull request merged**